app/main.py

```
@@ -1 +1,16 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.strip().split()
+
+    if len(parts) != 3 or parts[0] != "cp":
+        return
```

Issue: The checklist requires that the function should return an error message or handle the case when the command format is incorrect. Currently, it just returns None without any feedback. Consider providing an error message or raising an exception as per the checklist requirements.